src/helpers/event.ts

```diff
@@ -27,3 +27,14 @@ export function onError(emitter:EventEmitter): Promise<Error> {
     emitter.on('error', (e:Error) => resolve(e))
   })
 }
+
+/**
+ * The websocket enabled version of this method. Note that, in the `receipt` helper there
+ * is a function of a similar name, "eventsReturnValues", however, that method is meant to work with `TransactionReceipt` objects,
+ * while this one is for the `EventLog` returned from an `onData` emitter.
+ *
+ * Generally this is the preferred method, as you should be using websockets.
+ */
+export function eventReturnValues(name:string, log:EventLog): any {
```

Comment:
I feel like these names will be confusing since they are so similar

Reply:
the other will get removed as we should never be looking for events outside of the WS log. 

if we want to change -- we can change the original to something like receiptEventReturnValues or some shit